app/src/main/java/com/example/barterbuddy/activities/PersonalItemsDetailPage.java

```diff
@@ -82,7 +84,7 @@ protected void onCreate(Bundle savedInstanceState) {
                           .getReference()
                           .child("users/" + email + "/" + itemId + ".jpg");
 
-                  final long ONE_MEGABYTE = 1024 * 1024;
+                  final long ONE_MEGABYTE = 1024 * 1024 * 5;
```

Comment:
Why is this multiplied by 5 now but not named FIVE_MEGABYTES? This change should be explained, especially since the AdjustTradeMoney page still uses 1024 * 1024 for ONE_MEGABYTE